src/plugins/mer/meremulatordevicewidget.cpp

```diff
@@ -72,14 +76,31 @@ MerEmulatorDeviceWidget::~MerEmulatorDeviceWidget()
     delete m_ui;
 }
 
+void MerEmulatorDeviceWidget::onDeviceModelEdited()
+{
+    MerSettings::populateWithDeviceModels(m_ui->deviceModelComboBox);
```

Comment:
I would consider encapsulating the related code into class MerDeviceModelComboBox.